src/app/client/src/app/modules/workspace/components/all-content/all-content.component.ts

```diff
@@ -432,7 +432,20 @@ export class AllContentComponent extends WorkSpace implements OnInit, AfterViewI
   }
 
   contentClick(content) {
-    if (_.size(content.lockInfo)) {
+    if(content.originData){
+    var neww = JSON.parse(content.originData);
+    }
+    if(content.originData){
+      if((neww.copyType === 'shallow')){
+        this.toasterService.error('This is an adopted textbook from' + neww.organisation[0] + '.Hence it cannot be edited');
```

Comment:
@prashantsingh03 resourebundle string?

Reply:
and it should be like this.. This is an adopted textbook from' + neww.organisation[0] + '. Hence it cannot be edited

Reply:
@prashantsingh03     if (content.originData){
      if((neww.copyType === 'shallow'))  instead of this, 
if (content.originData){ 
       let tempOrigin = JSON.parse(content.originData);
      if((tempOrigin.copyType === 'shallow')){
                 this.toasterService.error('This is an adopted textbook from' + neww.organisation[0] + '.Hence it cannot be edited');
     return;
  }
}